apiserver/pkg/chat/chat_server.go

```diff
@@ -403,3 +404,46 @@ func (cs *ChatServer) GetApp(ctx context.Context, appName, appNamespace string)
 }
 
 // todo Reuse the flow without having to rebuild req same, not finish, Flow doesn't start with/contain nodes that depend on incomingInput.question
```

Comment:
what is this line?

Reply:
historical todo

we should look for a way to keep the app's runtime in memory so that it can be called quickly